lib/bom.js

```diff
@@ -136,6 +136,68 @@ declare class PluginArray {
     [key: number | string]: Plugin;
 }
 
+declare class PerformanceTiming {
+    connectEnd: number;
+    connectStart: number;
+    domainLookupEnd: number;
+    domainLookupStart: number;
+    domComplete: number;
+    domContentLoadedEventEnd: number;
+    domContentLoadedEventStart: number;
+    domInteractive: number;
+    domLoading: number;
+    fetchStart: number;
+    loadEventEnd: number;
+    loadEventStart: number;
+    navigationStart: number;
+    redirectEnd: number;
+    redirectStart: number;
+    requestStart: number;
+    responseEnd: number;
+    responseStart: number;
+    secureConnectionStart: number;
+    unloadEventEnd: number;
+    unloadEventStart: number;
+}
+
+declare class PerformanceNavigation {
+    type: 0 | 1 | 2 | 255;
```

Comment:
Should the constants also be added?
https://www.w3.org/TR/navigation-timing/#sec-navigation-info-interface

e.g.
```js
if (performance.navigation.type === performance.navigation.TYPE_BACK_FORWARD) doSomethingWhenReturningToPage()
```